### DISCUSSION TITLE
Удаление циклов через кнопку, и редактирование цикла из временной таблицы.

### DISCUSSION THREAD

**Discussion by Xamius** [totumonline/totum-ru-issues-and-discussions#350] [Как это сделать?] [answered]
Добрый день.
1. Подскажите как создать кнопку для удаления цикла по ID и возвратом (переходом) страницу во все циклы.
2. Можно ли изменять данные в циклов из временной таблицы.
3. Скрывать поля в футере или хэдере по чекбоксу? (Пока не поставил чекбокс поле не отображается)? 
Спасибо

**Comment by erpic-ru** [comment, marked as answer]
1. В коде кнопки:
```

a1=: delete(table...
a2=: linktotable(table: Таблица циклов  target: "self")

```
2. Да: через код действия при изменении и set

3. [Да, можно](https://github.com/totumonline/totum-ru-issues-and-discussions/discussions/142)
